src/EditCell.tsx

```diff
@@ -93,9 +93,22 @@ export default function EditCell<R, SR>({
   function handleKeyDown(event: React.KeyboardEvent<HTMLDivElement>) {
     if (onKeyDown) {
       const cellEvent = createCellEvent(event);
-      onKeyDown({ mode: 'EDIT', row, column, rowIdx, onClose }, cellEvent);
+      onKeyDown(
+        {
+          mode: 'EDIT',
+          row,
+          column,
+          rowIdx,
+          navigate() {
```

Comment:
Currently there is no way to continue navigation if default event is prevented. We need to pass this function to allow users to control navigation